package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac

```diff
@@ -61,6 +61,8 @@ elseif platform.match('mpc85xx', 'p1020', {'ocedo,panda'}) then
   table.insert(try_files, 1, '/sys/class/net/eth1/address')
 elseif platform.match('ramips', 'mt7620', {'miwifi-mini'}) then
   table.insert(try_files, 1, '/sys/class/net/eth0/address')
+elseif platform.match('ramips', 'mt7620', {'tplink,c20-v1'}) then
```

Comment:
This condition should be unified with the one above.

Reply:
Like now?

Reply:
Yes, except for the spacing. Check how it's done above.